src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -1780,6 +1781,38 @@ public ItemStack get(PlayerStonecutterRecipeSelectEvent event) {
 				}
 			}, EventValues.TIME_NOW);
 
+		// InventoryMoveItemEvent
```

Comment:
I think it might be better for the Source to be TIME_NOW and the Destination to be TIME_FUTURE since InventoryMoveItemEvent is cancellable.

Reply:
@UnderscoreTud suggested this change in https://github.com/SkriptLang/Skript/pull/5462#discussion_r1111067359